cgi/kratos_auth.pl

```diff
@@ -45,27 +47,20 @@
     my $resp = $ua->request($req);
 
     if ($resp->is_success) {
+        #decode json to a hash
         my $json = $resp->decoded_content;
-        log->debug($json);
+        my $content = decode_json($json);
+        my %contentdecoded = %$content;
```

Comment:
your names are a bit misleading.

```suggest
        my $content_ref = decode_json($json);
        my %content = %$content_ref;
```
Also you don't really need to do that. If you keep `$content_ref`, you can access data adding a `->`.
Like `$content_ref->{identity}{traits}{UserID}`

Reply:
Fixed, did not know you could do this. Thanks!